packages/web-react/src/hooks/useStyleUtilities.ts

```diff
@@ -0,0 +1,51 @@
+import { SpacingStyleProp } from '../constants';
+import { BreakpointToken, SpaceToken, StyleProps } from '../types';
+import { applyClassNamePrefix } from '../utils';
+
+export type StyleUtilitiesResult = {
+  styleUtilities: string[];
+  props: StyleProps;
+};
+
+const normalizeSpacingValue = (value: string): string => (value === 'auto' ? 'auto' : value.replace(/[^0-9]/g, ''));
+
+const processBreakpointProperties = (
```

Comment:
No tests for the unexported functions…?

I think tests are helpful in this case. They help understand what the function does (so one doesn't have to google `/[^0-9]/g`), makes advanced modifications like this more reliable, and – most importantly – improves our code coverage badge 🙂.

Reply:
No, we don't want to export them, as we don't want to keep them under semver. And without export I am not able to test it.